src/DAL/Events/Repositories/EventSeatRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace DAL.Events.Repositories
+{
+	sealed class EventSeatRepository : IGenericRepository<EventSeat, long>
+	{
+		private readonly EventContext _context;
+
+		public EventSeatRepository(EventContext context)
```

Comment:
Looks like you have similar boilerplate code for all entities. You have generic interface. Maybe you could implement generic Base class with all the common CRUD operations, and remove all these duplicated implementations in each class?

Reply:
Sure, with a bit of Reflection magic.